docs/source/spec/core/stream-traits.rst

```diff
@@ -1,191 +1,100 @@
+        operation StreamingOperation {
+            output: StreamingOutputWrapper,
         }
 
-        structure SubscribeToMovementsOutput {
-            @eventStream
-            movements: Movement,
+        structure StreamingOutputWrapper {
+            output: StreamingBlob,
         }
 
-        structure Movement {
-            angle: Float,
-            velocity: Float,
-        }
-
-    .. code-tab:: json
+        @streaming
+        blob StreamingBlob
 
-        {
-            "smithy": "1.0.0",
-            "shapes": {
-                "smithy.example#SubscribeToMovements": {
-                    "type": "operation",
-                    "output": {
-                        "target": "smithy.example#SubscribeToMovementsOutput"
-                    }
-                },
-                "smithy.example#SubscribeToMovementsOutput": {
-                    "type": "structure",
-                    "members": {
-                        "movements": {
-                            "target": "smithy.example#Movement",
-                            "traits": {
-                                "smithy.api#eventStream": true
-                            }
-                        }
-                    }
-                },
-                "smithy.example#Movement": {
-                    "type": "structure",
-                    "members": {
-                        "angle": {
-                            "target": "smithy.api#Float"
-                        },
-                        "velocity": {
-                            "target": "smithy.api#Float"
-                        }
-                    }
-                }
-            }
-        }
-
-The name of the event sent over a single-event event stream is the name
-of the member that is targeted by the ``eventStream`` trait.
-
-
-Single-event client behavior
-============================
-
-Clients that send or receive single-event event streams are expected to
-provide an abstraction to end-users that allows values to be produced or
-consumed asynchronously for the targeted event structure. Because a
-single-event event stream does not utilize named events like a multi-event
-event stream, functionality used to dispatch based on named events is
-unnecessary. Clients MUST provide access to the
-:ref:`initial-message <initial-messages>` of an event stream when necessary.
+.. _event-streams:
 
+-------------
+Event Streams
+-------------
 
-.. _multi-event-event-stream:
+An event stream is an abstraction that allows multiple messages to be sent
+asynchronously between a client and server. Event streams support both duplex
+and simplex streaming. The serialization format and framing of messages sent
+over event streams is defined by the :ref:`protocol <protocolDefinition-trait>`
+of a service.
 
--------------------------
-Multi-event event streams
--------------------------
+An operation can send an event stream as part of its input or output. An
```

Comment:
Seems like the first sentence can be removed, this paragraph can be combined with the next one.